flink-runtime/src/main/java/org/apache/flink/runtime/state/CheckpointListener.java

```diff
@@ -18,11 +18,14 @@
 package org.apache.flink.runtime.state;
 
 
+import org.apache.flink.annotation.PublicEvolving;
+
 /**
  * This interface must be implemented by functions/operations that want to receive
  * a commit notification once a checkpoint has been completely acknowledged by all
  * participants.
  */
+@PublicEvolving
```

Comment:
Separate commit?

Reply:
Ideally yes, but I kept it because there is not a clear commit where it should belong to and I find it too trivial for a separate commit.